src/Engine/Systems/RenderSystem.cpp

```
@@ -64,13 +65,38 @@ void RenderSystem::init(const Geometry::GeometryData *geomData, const Ctx::Offsc
 
 	m_sessionResourceID = m_cleanupManager->createCleanupGroup();
 
+	initOrbitVertexArray();
 	initGlobalBuffers();
 	createVisualizers();
 	initVisualizerCmdBufSets();
 	initVisualizers();
 }
 
 
+void RenderSystem::initOrbitVertexArray() {
```

Clang (14.0.6)

[warning] method 'initOrbitVertexArray' can be made static

(readability-convert-member-functions-to-static)